utils/singularity/def/capice-3.0.0rc2.def

```diff
@@ -0,0 +1,16 @@
+Bootstrap: library
+From: ubuntu:20.04
```

Comment:
Ideally image should derive from same Alpine Linux parent as all other containers.

Reply:
https://github.com/molgenis/vip/issues/232